src/inputs/internal/ComboBoxBase.tsx

```diff
@@ -212,9 +219,9 @@ export function ComboBoxBase<O, V extends Value>(props: ComboBoxBaseProps<O, V>)
   }
 
   async function maybeInitLoad() {
-    if (!Array.isArray(maybeOptions)) {
+    if (!Array.isArray(propOptions)) {
       setFieldState((prevState) => ({ ...prevState, optionsLoading: true }));
-      await maybeOptions.load();
+      await propOptions.load();
```

Comment:
This was the one place that needed to go back to the `current/load/options` to get the `load` function